compiler/control/OMROptions.cpp

```diff
@@ -1612,6 +1612,7 @@ int32_t       OMR::Options::_maxNumVisitedSubclasses = 500;
 
 int32_t       OMR::Options::_minProfiledCheckcastFrequency = 20; // as a percentage
 int32_t       OMR::Options::_lowCodeCacheThreshold = 256*1024; // 256KB Turn off Iprofiler if available code cache space is lower than this value
+int32_t       OMR::Options::_highCodeCacheOccupancyPercentage = 75;
```

Comment:
Make sure we perform a validation of this option when the user specifies a value: it should be in the 0-100 range.
Also, let's add a comment here about that.